kafka-ui-e2e-checks/src/test/java/com/provectus/kafka/ui/tests/TopicTests.java

```diff
@@ -64,9 +62,9 @@ public void createTopic() {
     @CaseId(197)
     @Test
     public void updateTopic() {
-        pages.openTopicsList(CLUSTER_NAME)
+        topicsList.goTo(CLUSTER_NAME)
```

Comment:
we don't need to remove navigation steps from test. please return mainPage.goTo()
.goToSideMenu(CLUSTER_NAME, bla-bla